build.rs

```diff
@@ -0,0 +1,16 @@
+use std::process::Command;
+
+/* Prebuild script */
+fn main() {
+    println!("cargo:rerun-if-env-changed=ALWAYS_RUN");
+
+    // Create a new command that runs bash
+    let mut command = Command::new("python3");
```

Comment:
Can the path to the python3 binary optionally be an environment variable?  Just thinking as buildroot doesn't like accessing host binaries that aren't "required".

Reply:
Eh don't worry about this I will add ruamel.yaml to the docker and just require people have it on host.  Building it is probabaly overkill